components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -58,7 +62,45 @@ export default class AppsFormSelectField extends React.PureComponent<Props, Stat
         return null;
     }
 
-    onChange = (selectedOption: AppSelectOption) => {
+    getCommonProps = (): Partial<AsyncSelectProps<AppSelectOption>> => {
+        const {field} = this.props;
+        const placeholder = field.hint || '';
+        const value = this.props.value;
+
+        return {
+            value,
+            placeholder,
+            isDisabled: field.readonly,
+            onChange: this.onChange as any, // types are not working correctly for multiselect
+            isMulti: field.multiselect || false,
+            isClearable: true,
+            openMenuOnFocus: false,
+            classNamePrefix: 'react-select-auto react-select',
+            menuPortalTarget: document.body,
+            styles: reactStyles,
+            menuPlacement: 'bottom',
```

Comment:
@mickmister I'm not exactly sure how this works, but through a quick google search, it looks like there is a `menuPlacement: 'auto'` option. I assume that this will place the menu options above or below depending on the available real estate. Or did you try this already?